### ISSUE TITLE
New API key not working

### ISSUE THREAD

**Issue by JJHontebeyrie** [Bodmer/OpenWeather#59]
Seems new (recent) API key are no more working with this library

**Comment by Bodmer** [comment]
Thanks for reporting this.

OpenWeather uses the OneCall API and I see this has now been changed so it is necessary to signup to a subscription.  1000 calls per day are still free. As you found, the older API keys do still work but new ones do not.

The remaining "free" weather API calls have a different format and thus won't be decoded at all.

I will have to think on this further as a significant update to the library will be required to accomdate the alternative API responses... and that will take some time!

**Comment by JJHontebeyrie** [comment]
Thanks for the answer. But if this library works with the One Call by Call subscription it is ok. 1000 calls per day is enought for most electronic applications at home.

**Comment by Bodmer** [comment]
Yes, it will work and 1000 calls per day is plenty. You do run the risk of being charged for each call over 1000 but it looks like you can set a call limit as part of the subscription package.

I think most library users will not want to sign up for a paid service though.

I will look at the free API responses and see if they provide the required data.

**Comment by Bodmer** [comment]
I have updated the library to use the "forecast" API rather than the "onecall" API. This means no subscription is needed (no payment details need to be given when subscribing ).

If you have time to test then that would be helpful.

The "forecast" API call returns less information than the "onecall" API, but it is sufficient for the TFT sketch. The basic serial only sketch has also been updated. These should both work fine with new API access keys.
